spec/support/authorization_view_helper.rb

```diff
@@ -33,4 +33,38 @@ def get_basic_editor_views(model_name, things_to_look_for)
       #page.should have_content(things_to_look_for[0].titlecase)
     end
   end
-end
\ No newline at end of file
+
+  def get_nested_editor_views(outer_model_name, nested_model_name, things_to_look_for)
+    outer_model = outer_model_name.to_sym
+    nested_model = nested_model_name.to_sym
+    nested_model_controller = nested_model_name.pluralize.to_sym
+
+    it "gets the index" do
+      # I was unable to get the later attributes call to work unless factorygirl call was within the test
+      @sample_object = create(nested_model)
+      index_model_path = url_for(action: :index, controller: nested_model_controller)
+      visit index_model_path
+      expect(page).to have_content("LIMS") # This is in the nav bar
```

Comment:
Not going to stop the PR for this, but we should be starting to filter these out.